targets/TARGET_STM/mbed_rtx.h

```diff
@@ -19,7 +19,21 @@
 
 #ifndef INITIAL_SP
 
-#if (defined(TARGET_STM32F051R8) ||\
+#if (defined(TARGET_STM32L475VG) ||\
+     defined(TARGET_STM32L476RG) ||\
+     defined(TARGET_STM32L476JG) ||\
+     defined(TARGET_STM32L476VG) ||\
+     defined(TARGET_STM32L486RG))
+/* only GCC_ARM and IAR toolchains have the stack on SRAM2 */
+#if (((defined(__GNUC__) && !defined(__CC_ARM)) ||\
```

Comment:
Why not for ARMCC? only those 2 toolchains are here supported?

Reply:
Hello @0xc0170 
IAR was done in #5844 
This PR is for GCC_ARM (required for Cloud example).
ARM will come soon.

Reply:
I assume will be created as a new PR that will address ARMCC support for this. I would like to avoid having different behavior between toolchains

Reply:
We will do it .